docs/api.rst

```diff
@@ -25,6 +25,5 @@
 API Docs
 ========
 
-invenio_deposit
----------------
-
+.. automodule:: invenio_deposit.api
```

Comment:
So, for my PR for `invenio-records-rest` I need before at least `invenio_search.api.RecordsSearch` is available online.